diff --git a/CMake/TdSetUpCompiler.cmake b/CMake/TdSetUpCompiler.cmake
index 8e84343..f6d0cb2 100644
--- a/CMake/TdSetUpCompiler.cmake
+++ b/CMake/TdSetUpCompiler.cmake
@@ -10,7 +10,7 @@ function(td_set_up_compiler)
   if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     set(GCC 1)
     set(GCC 1 PARENT_SCOPE)
-  elseif (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  elseif (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND NOT MSVC)
     set(CLANG 1)
     set(CLANG 1 PARENT_SCOPE)
   elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
