--- a/Makefile
+++ b/Makefile
@@ -240,6 +240,12 @@ else ifeq ($(os),visionOS)
 TARGET_TRIPLE-$(target)=$$(ARCH-$(target))-apple-xros
 else ifeq ($(os),visionOS-simulator)
 TARGET_TRIPLE-$(target)=$$(ARCH-$(target))-apple-xros-simulator
+else ifeq ($(os),iOS-simulator)
+TARGET_TRIPLE-$(target)=$$(ARCH-$(target))-apple-ios-simulator
+else ifeq ($(os),tvOS-simulator)
+TARGET_TRIPLE-$(target)=$$(ARCH-$(target))-apple-tvos-simulator
+else ifeq ($(os),watchOS-simulator)
+TARGET_TRIPLE-$(target)=$$(ARCH-$(target))-apple-watchos-simulator
 else
 	ifeq ($$(findstring simulator,$$(SDK-$(target))),)
 TARGET_TRIPLE-$(target)=$$(ARCH-$(target))-apple-$$(OS_LOWER-$(target))
